packages/core/react/src/scripts/postbuild.js

```diff
@@ -1,21 +1,78 @@
 /* eslint-disable no-console */
 const fs = require('fs');
+const path = require('path')
+
+const chalk = require('chalk');
+
+const coreDir = path.resolve(__dirname, '../..'); // Core directory path
+const packagesDir = path.resolve(coreDir, '../..'); // Packages directory path
+const componentsDir = path.resolve(packagesDir, 'components'); // Components directory path
+const outputPath = path.resolve(coreDir, 'dist'); // Output directory path
+
+const rootDir = path.resolve(__dirname, '../../../../..'); // Root directory path
+const appsConfigDir = path.resolve(rootDir, 'apps/docs/config'); // Apps config directory path
+const appsRoutesJsonPath = path.resolve(appsConfigDir, 'routes.json'); // Apps routes file path
```

Comment:
@winchesHe I think we could move this script to the root inside the `scripts` folder

Reply:
Got